TodoApp.Services/TodoService.cs

```diff
@@ -71,5 +69,10 @@ public bool DeleteByIndex(int index)
             bool isDeleted = this.Delete(modelId);
             return isDeleted;
         }
+
+        public bool HasTodos()
+        {
+            return repo.GetAll() != null;
```

Comment:
What happens if the repo returns an empty list of Todos?
```suggestion
            var todos = repo.GetAll();
            return todos != null && todos.Any();
```